ocaml/parsing/builtin_attributes.ml

```diff
@@ -478,7 +478,9 @@ let layout ~legacy_immediate attrs =
      | Immediate | Immediate64 ->
         check  (legacy_immediate
              || Language_extension.(is_at_least Layouts Beta))
-     | Any | Void | Float64 ->
+     | Float64 ->
+        check Language_extension.(is_at_least Layouts Beta)
+     | Any | Void ->
```

Comment:
I'm confused - my impression was you explicitly left out any from this in the previous PR that moved the rest of `any` to beta because you encountered some issues.  Is that not right?

Reply:
I don't recall that. Maybe you're thinking about the fact that I want to delete this function entirely? Which is true, but it's easy enough to make this small improvement in the meantime. So the problem is likely originally my oversight -- sorry.

Reply:
OK, I took a look at this and don't see any fundamental obstacles, but there are a bunch more test CRs to address when we do it, so I suggest it should be its own PR.